hphp/runtime/vm/jit/unique-stubs.cpp

```diff
@@ -1118,12 +1123,26 @@ TCA emitEnterTCHelper(CodeBlock& cb, DataBlock& data, UniqueStubs& us) {
 #endif
 
   return vwrap2(cb, cb, data, [&] (Vout& v, Vout& vc) {
-    // Native func prologue.
-    v << stublogue{true};
+    bool saveFrame = true;
 
     // Architecture-specific setup for entering the TC.
     v << inittc{};
 
+    // Set if FP need to be saved.
+    switch (arch()) {
+      case Arch::ARM:
+      case Arch::X64:
+        break;
+      case Arch::PPC64:
+        // Do no save the FP for PPC64, as no new frame is created in the
+        // stublogue
+        saveFrame = false;
+        break;
+    }
+
+    // Native func prologue.
+    v << stublogue{saveFrame};
```

Comment:
Hmm... what's the point of having `stublogue{true}` on PPC64 in `emitFunctionEnterHelper()`, then?